content/_data/roadmap/roadmap.yml

```diff
@@ -43,13 +43,18 @@ categories:
       status: future
   - name: Packaging and platform support
     initiatives:
-    - name: OpenJ9 distributions
+    - name: Docker images for IBM s390x
       status: current
-    - name: Docker images for Arm64
+      description: Docker image support for IBM series 390 mainframes running Java 8 and Java 11
```

Comment:
Too long for the current layout

Reply:
That's the description which is shown in hover text.  Long descriptions seem to fit when I used them even at 800x600.

![image](https://user-images.githubusercontent.com/156685/78272160-62eaeb00-74ca-11ea-90e8-bd79ca01a220.png)